<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArmSim</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/resources/logo.png</normaloff>:/resources/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="6,6,1,4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
      <item>
       <widget class="MemoryWidget" name="memoryWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="RegistersWidget" name="registersWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
      <item>
       <widget class="DisassemblyWidget" name="disassemblyWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="StackWidget" name="stackWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,1">
      <item>
       <widget class="RunControlsWidget" name="runControlsWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="FlagsWidget" name="flagsWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
      <item>
       <widget class="TerminalWidget" name="terminalWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="LoaderWidget" name="loaderWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LoaderWidget</class>
   <extends>QWidget</extends>
   <header>ui/loaderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MemoryWidget</class>
   <extends>QWidget</extends>
   <header>ui/memorywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TerminalWidget</class>
   <extends>QWidget</extends>
   <header>ui/terminalwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FlagsWidget</class>
   <extends>QWidget</extends>
   <header>ui/flagswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StackWidget</class>
   <extends>QWidget</extends>
   <header>ui/stackwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RegistersWidget</class>
   <extends>QWidget</extends>
   <header>ui/registerswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DisassemblyWidget</class>
   <extends>QWidget</extends>
   <header>ui/disassemblywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RunControlsWidget</class>
   <extends>QWidget</extends>
   <header>ui/runcontrolswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
